integration/test/empty/empty_test.go

```diff
@@ -0,0 +1,7 @@
+package empty
+
+import "testing"
+
+func TestEmpty(t *testing.T) {
```

Comment:
What's this for?

Reply:
I like to have an empty test as a sanity check to make sure there is no issue with starting / running tests

Reply:
Will add comment

Reply:
Maybe this is just a bad habit of mine from my spring days where application context was a bigger deal for integration tests. As go does not have bean wiring.

Reply:
You're free from the clutches of Spring now. There's nothing to `@Autowire` anymore. Let go. 😃 

I'm not opposed to having it if we include a comment why it's here. It just popped out to me as odd when I saw it during review.